src/commands.cpp

```diff
@@ -3377,6 +3384,13 @@ void CommandGetUserData::procresult()
             }
             break;
 
+        case MAKENAMEID4('s', 'm', 's', 'v'):
+            if (!client->json.storeobject(&smsv))
```

Comment:
Why are we parsing this parameter?? it's used nowhere. Does `smsv` value represent the phone-number configured by the user for SMS verification? I don't know. However, the API command `smsv` is used to verify a received code, so it's strange to me to see it in the `CommandGetUserData`

Reply:
Just read the `smsv` will be the phone-number you have verified. Since this parameter is not used and unrelated to this pull-request, it would be better to skip it. Same applies to the `smsve` inside the `flags` array.

Reply:
thanks for spotting that; I've removed those in this PR.  cheers